tests/test_unit.py

```diff
@@ -538,5 +539,82 @@ def test_group_by_package(self):
         self.assertCountEqual(toc_yaml_got, toc_yaml_want)
 
 
+
+    def test_check_name_with_uid(self):
+        # Test that none of these entries change.
+        toc_yaml_want = [
+            {
+                "name":"Overview",
+                "uid":"google.cloud.spanner.types"
+            },
+            {
+                "name":"types",
+                "uid":"google.cloud.spanner.types"
```

Comment:
Two TOC entries with the same `uid`?

Reply:
interestingly, they do appear duplicated in the actual toc:
```
    - items:
      - name: Overview
        uid: google.cloud.logging_v2.client
      - name: Client
        uid: google.cloud.logging_v2.client.Client
      name: client
      uid: google.cloud.logging_v2.client
```

I'll update this test case however to avoid confusion.